packages/mailchimp/test/gzip.test.js

```diff
@@ -0,0 +1,87 @@
+import { expect } from 'chai';
+import * as undici from 'undici'
+import {Readable } from 'node:stream';
+import tar from 'tar-stream';
+import fs from 'node:fs'
+import gunzip from 'gunzip-maybe';
+import fixtureData from './fixtures/data.json' assert { type: 'json' }
+
+describe('gzip', () => {
+  it('should extract a tarball as json', (cb) => {
```

Comment:
This test shows how to use the streaming API to parse a tar file (not gzipped!). I had to start here to understand the principe.

This test isn't actually helpful and should be removed.